NN_chatCmd.js

```diff
@@ -12,17 +13,28 @@ nerthus.chatCmd.run = function(ch)
         var callback = this.fetch_callback(cmd,ch)
         if(callback)
         {
-            log("["+ch.k+"] " + ch.n + " -> " + ch.t) //gdze kto co
-            return callback(ch)
+            log("["+ch.k+"] " + ch.n + " -> " + ch.t +
+            " (" + nerthus.chatCmd.current_map +")"); //gdzie kto co mapa
+            return callback(ch);
         }
     }
 }
 
 nerthus.chatCmd.fetch_cmd = function(ch)
 {
-    if(ch.t[0]=='*')
-        return RegExp(/^\*(\S+)/).exec(ch.t)
-    return null
+    if (nerthus.chatCmd.currentMap === '')
```

Comment:
To nie zadziała, cały obiekt nerthus jest zapisywany w localStorage co oznacza, że jak raz się ustawi currentMap i dojdzie do zapisu nerthusa (np. w wyniku zassania nowej wersji dodatku, zmiany ustawień) to wartość currentMap już nigdy się nie zmieni i to wszystko nie będzie działać. Jeśli już używać jakiejś zmiennej to nie może ona być w obiekcie nerthus.